amazoncorretto/Dockerfile

```diff
@@ -0,0 +1,90 @@
+#
+# Scala and sbt Dockerfile
+#
+# https://github.com/sbt/docker-sbt
+#
+
+# Pull base image
+ARG BASE_IMAGE_TAG
+FROM amazoncorretto:${BASE_IMAGE_TAG:-21.0.5-al2023}
+
+# Env variables
+ARG SCALA_VERSION
+ENV SCALA_VERSION=${SCALA_VERSION:-3.3.4}
+ARG SBT_VERSION
+ENV SBT_VERSION=${SBT_VERSION:-1.10.5}
+ARG USER_ID
+ENV USER_ID=${USER_ID:-1001}
+ARG GROUP_ID
+ENV GROUP_ID=${GROUP_ID:-1001}
+
+# Install git and rpm for sbt-native-packager (see https://github.com/sbt/docker-sbt/pull/114)
+RUN \
+  dnf -y update && \
+  dnf -y install tar gzip procps git rpm && \
```

Comment:
Adding procps for ps.

https://github.com/sbt/docker-sbt/pull/207#issuecomment-2476388865

Reply:
Will apply that to graalvm after the merge. Thanks